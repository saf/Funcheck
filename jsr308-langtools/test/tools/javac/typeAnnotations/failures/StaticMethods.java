/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary static methods don't have receivers
 * @author Mahmood Ali
 * @compile/fail/ref=StaticMethods.out -XDrawDiagnostics -source 1.7 StaticMethods.java
 */
class StaticMethods {
  static void main() @A { }
}

@interface A { }
