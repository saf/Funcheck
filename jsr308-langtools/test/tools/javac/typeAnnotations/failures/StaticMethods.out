StaticMethods.java:9:22: compiler.err.annotation.type.not.applicable
1 error
